shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-opengauss/pom.xml

```diff
@@ -40,26 +40,4 @@
             <version>${project.version}</version>
         </dependency>
     </dependencies>
-    
```

Comment:
@pk-218 Why delete this profile?

Reply:
I had a merge conflict when I had made this PR in the `PostgreSQLStatementParser` grammar. Hence I pulled changes from master, rebased with the branch I was working on,and these changes related to the PR #16306 had to be added to my working branch.

Reply:
@strongduanmu Do you suggest any fix for this?